var/spack/repos/builtin/packages/hepmcanalysis/package.py

```diff
@@ -22,17 +22,17 @@ class Hepmcanalysis(MakefilePackage):
     depends_on('root')
     depends_on('clhep')
 
-    variant('cxxstd',
-            default='11',
-            values=('11', '14', '17'),
-            multi=False,
-            description='Use the specified C++ standard when building.')
-
     patch('lcg.patch')
 
+    @run_before('build')
```

Comment:
```suggestion

```
`patch` is a spack stage that always runs before build

Reply:
I agree that it was not necessary looking back at the build output in sft-spack-repo. However, if patch always runs before build, I think it should be specified better in the documentation: https://spack.readthedocs.io/en/latest/packaging_guide.html#patch-functions: "A patch function, if present, will be run after patch files are applied and before install() is run."